examples/nextjs/pages/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { Sandpack } from "@codesandbox/sandpack-react";
+import { Sandpack } from "@codesandbox/sandpack-react/unstyled";
```

Comment:
Unfortunately, types are still failing here. @Jokcy would you like to give it a try?

Reply:
Sure, did not notice the types, will check later,

Reply:
@danilowoz Based on https://github.com/microsoft/TypeScript/issues/50794, set `moduleResolution` to `node16` will fix the types.

Reply:
Great! I spent quite some time on this issue, and I couldn't find a solution. Cool!